registry-disk.md

```diff
@@ -0,0 +1,77 @@
+## Registry Disk
+
+The Registry Disk feature provides the ability to store and distribute Virtual
+Machine disks in the container image registry. Registry Disks can be assigned
+to Virtual Machines in the disks section of the Virtual Machine spec.
+
+## When to use a Registry Disk
+
+Registry Disks are ephemeral storage devices that can be assigned to any number
+of active Virtual Machines. This makes them an ideal tool for users who want
+to replicate a large number of Virtual Machine workloads that do not require
+persistent data. Registry Disks are commonly used in conjunction with Virtual
+Machine Replica Sets.
+
+## When Not to use a Registry Disk
+
+Registry Disks are not a good solution for any workload that requires persistent
+disks across Virtual Machine restarts, or workloads that require Virtual
```

Comment:
Regarding to restarts, if it is just a restart on the same node, will it keep the data?

Reply:
nope, the only case where data will be reused is if the process was allowed to recover within the same virt-launcher cgroup.